test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestProjectContext.cs

```diff
@@ -276,6 +275,14 @@ public void AddPackageToAllFrameworks(params SimpleTestPackageContext[] packages
             }
         }
 
+        public void CleanPackagesFromAllFrameworks()
+        {
+            foreach (var framework in Frameworks)
+            {
+                framework.PackageReferences = new List<SimpleTestPackageContext>();
```

Comment:
```suggestion
                framework.PackageReferences.Clear();
```

This minimises memory allocation

Reply:
Good suggestion! Updated.